### ISSUE TITLE
Interface wl_surface has no event

### ISSUE THREAD

**Issue by Tahinli** [GyulyVGC/sniffnet#607]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### What's the problem?

I tried to open Sniffnet

### How did you install the app?

downloaded one of the provided packages

### Operating System

Linux

### Additional context

![image](https://github.com/user-attachments/assets/9b86749f-ca7f-4dda-af9b-6570f514465a)

My System:
10750H
1660ti Proprietary Driver
Fedora 40 KDE Wayland

**Comment by GyulyVGC** [comment]
Hi @Tahinli, you have downloaded 1.2.2, an older version of the app.

This issue was solved with newer versions of the application.

I suggest you to download the latest available version.

**Comment by Tahinli** [comment]
> This issue was solved with newer versions of the application.

Sorry I've downloaded Sniffnet while ago. I thought it's controlled by Fedora official repos, I forget I installed it from external package.

Btw any chance to push Sniffnet to official distro repos ?

**Comment by GyulyVGC** [comment]
At the moment Sniffnet is on the official repositories of Arch, NetBSD, FreeBSD, NixOS, and Tiny Core Linux.

I didn't judge it a priority to also put it in the official Debian and Fedora repos since I'm already distributing the DEB and RPM packages.
I tried to package it for Debian but I noticed this was requiring me too much effort as many dependencies of Sniffnet were missing a source package.
Anyway this is something that I could consider for the future.
